src/Resources/config/grids.yml

```diff
@@ -0,0 +1,26 @@
+sylius_grid:
+  grids:
+    app_admin_subscription:
+      driver:
+        options:
+          class: "%app.model.shipping_subscription.class%"
+      fields:
+        customer:
+          type: string
+          label: sylius.ui.customer
+        enabled:
+          type: twig
+          label: sylius.ui.enabled
+          options:
+            template:
+              "@BitBagSyliusShippingSubscriptionPlugin/Grid/enabled.html.twig"
+        order_item_unit.order_item.product_name:
+          type: string
+          label: sylius.ui.product
+        created_at:
+          type: datetime
+          label: sylius.ui.from
+      actions:
+        item:
+          delete:
+            type: delete
```

Comment:
You need to follow project standards if there are 4 spaces in `yaml` files. You have 2 spaces indention in all cases.